db/schema.rb

```diff
@@ -827,21 +827,21 @@
     t.index ["replaced_by_id"], name: "index_global_meter_attributes_on_replaced_by_id"
   end
 
-  create_table "good_job_processes", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+  create_table "good_job_processes", id: :uuid, default: -> { "public.gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
This doesn't seem to be part of this change? I noticed this had popped up after running latest migrations locally

Reply:
Not sure why that changed - some version thing perhaps - I've reverted it to be on the safe side..